.github/workflows/ci.yml

```diff
@@ -5,10 +5,16 @@ jobs:
     runs-on: macos
     steps:
     - uses: actions/checkout@v2
-    - name: Run Tests
+    - name: Run Tests with Xcode
       run: |
         DESTINATION="platform=iOS Simulator,name=iPhone 8" SCHEME="Puree"
         xcodebuild test -project Puree.xcodeproj -scheme "${SCHEME}" -destination "${DESTINATION}"
+  SwiftPM:
+    runs-on: macos
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run Tests with Swift Package Manager
+      run: swift test -Xswiftc "-target" -Xswiftc "x86_64-apple-macosx10.12"
```

Comment:
The package manager currently hardcodes the deployment target to macOS 10.10, So need to overwrite the target to pass the test. (I don't know why it doesn't work even though we've specified iOS as a supported platform ...)

Reply:
According to the documentation, we must specify macOS platform version in current limitation...

 https://github.com/apple/swift-package-manager/blob/03c10bed07709f6b1b2787433ed15001f7ce433c/Sources/PackageDescription/SupportedPlatforms.swift#L52-L66